python/orca/tutorial/NCF/pytorch_model.py

```diff
@@ -23,7 +23,8 @@
 
 class NCF(nn.Module):
     def __init__(self, user_num, item_num, factor_num, num_layers,
-                 dropout, model, GMF_model=None, MLP_model=None):
+                 dropout, model, GMF_model=None, MLP_model=None,
+                 sparse_feats_dims=0, num_dense_feats=0):
```

Comment:
the default value shouldn't be 0? Make the default value to be None? If it is None, make it an empty list in init?